metaci/repository/models.py

```diff
@@ -57,10 +57,17 @@ def get_github_api(self):
 
     @property
     def latest_release(self):
+        from metaci.release.models import Release
+
         try:
-            return self.releases.latest()
-        except Repository.DoesNotExist:
-            return None
+            return self.releases.latest()  # Also for MetaCI create release button.
+        except Release.DoesNotExist:
+            Release(
+                repo=self,
+                version_name="1.0",
+                version_number=1.0,
+                git_tag=f"{self.release_tag_regex}1.0",
+            )
```

Comment:
@jofsky I'm not sure what you're trying to do here. You're instantiating a Release without saving it or returning it. I think you could leave it as `return None` and the important part of the fix is changing from Repository.DoesNotExist to Release.DoesNotExist, right?